src/tests/Microsoft.PowerFx.Core.Tests/PrimitiveValueConversionsTests.cs

```diff
@@ -260,7 +260,6 @@ public override void Visit(ITypeVisitor vistor)
         [InlineData("D", "T", "UnaryOpKind.DateToTime")] // new
         [InlineData("D", "d", "UnaryOpKind.DateToDateTime")] // new
         [InlineData("n", "$", "UnaryOpKind.NumberToCurrency")] // new
-        [InlineData("$", "n", "UnaryOpKind.CurrencyToNumber")] // new
```

Comment:
>  [InlineData("$", "n", "UnaryOpKind.CurrencyToNumber")]  [](http://example.com/codeflow?start=6&length=57)

We should still test this conversion. I believe the result (coercionKindStr) now should be function:Float